logtail/logtail.go

```diff
@@ -137,6 +160,10 @@ type Logger struct {
 	metricsDelta   func() string // or nil
 	privateID      PrivateID
 
+	instanceKey             *string
+	includeInstanceSequence bool
+	instanceSequence        int64
```

Comment:
Thinking about it more, I'm okay with this being an `uint64`. It will take a very long time (i.e., 2^53 messages) for it to be indistinguishable as a `float64`. Also, when the process first starts, the number will be small. As it grows bigger, the leading prefix will have the same bytes, and thus compress well.

Reply:
Switched to `uint64`